Mixpanel/Controller.cs

```diff
@@ -345,6 +378,9 @@ internal static void DoEngage(Value properties)
             properties["$mp_metadata"] = Metadata.GetPeopleMetadata();
 
             MixpanelStorage.EnqueueTrackingData(properties, MixpanelStorage.FlushType.PEOPLE);
+            if (Debug.isDebugBuild) {
+                MixpanelStorage.HasUsedPeople = true;
```

Comment:
no automatic people updates to check for on Unity?

Reply:
there are no automatic people updates in unity